src/bivariate_normals.jl

```diff
@@ -326,7 +326,7 @@ Note that nubar is not included.
 """
 function GalaxySigmaDerivs{NumType <: Number}(
     e_angle::NumType, e_axis::NumType, e_scale::NumType,
-    XiXi::SMatrix{2,2,NumType,4}, nuBar::NumType = one(NumType), calculate_tensor::Bool=true)
+    XiXi::SMatrix{2,2,NumType,4}, nuBar::Float64=1.0, calculate_tensor::Bool=true)
```

Comment:
Ah that's not the issue. The compiler decided to insert an optimization break. That gets fixed by making the other signature more aggressive and preventing and error when NumType is dual (because nuBar was still Float64).